android/app/src/main/AndroidManifest.xml

```diff
@@ -1,81 +1,154 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:tools="http://schemas.android.com/tools"
-  package="com.expensify.chat">
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.expensify.chat">
 
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.CAMERA" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
 
     <!-- android:hardwareAccelerated is essential for Android performance: https://developer.android.com/topic/performance/hardware-accel -->
     <application
-      android:supportsRtl="false"
-      android:hardwareAccelerated="true"
-      android:name=".MainApplication"
-      android:label="@string/app_name"
-      android:icon="@mipmap/ic_launcher"
-      android:roundIcon="@mipmap/ic_launcher_round"
-      android:allowBackup="false"
-      android:resizeableActivity="false"
-      android:theme="@style/AppTheme"
-      tools:replace="android:supportsRtl">
-
-      <activity
-        android:name=".MainActivity"
```

Comment:
Most of the manifest changes are fixing bad formatting. I will comment to point out the actual changes.